src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1802 @@
+        ),
+        "ce_ever_diagnosed": Property(
+            Types.BOOL,
+            "ever diagnosed with cervical cancer (even if now cured)"
+        ),
+        "ce_new_stage_this_month": Property(
+            Types.BOOL,
+            "new_stage_this month"
+        ),
+        "ce_xpert_hpv_ever_pos": Property(
+            Types.BOOL,
+            "hpv positive on xpert test ever"
+        ),
+        "ce_via_cin_ever_detected": Property(
+            Types.BOOL,
+        "cin ever_detected on via"
+        ),
+        "ce_date_last_screened": Property(
+          Types.DATE,
+          "date of last screening"
+        ),
+        "ce_date_thermoabl": Property(
+            Types.DATE,
+        "date of thermoablation for CIN"
+        ),
+        "ce_date_cryotherapy": Property(
+            Types.DATE,
+            "date of cryotherapy for CIN"
+        ),
+        "ce_date_via": Property(
+            Types.DATE,
+            "date of VIA screening"
+        ),
+        "ce_date_xpert": Property(
+            Types.DATE,
+            "date of XPERT screening"
+        ),
+        "ce_current_cc_diagnosed": Property(
+            Types.BOOL,
+            "currently has diagnosed cervical cancer (which until now has not been cured)"
+        ),
+        "ce_selected_for_via_this_month": Property(
+            Types.BOOL,
+            "selected for via this period"
+        ),
+        "ce_selected_for_xpert_this_month": Property(
+            Types.BOOL,
+            "selected for xpert this month"
+        ),
+        "ce_biopsy": Property(
+            Types.BOOL,
+            "ce biopsy done"
+        )
+    }
+
+    def read_parameters(self, data_folder):
+        """Setup parameters used by the module, now including disability weights"""
+
+        # Update parameters from the resourcefile
+        self.load_parameters_from_dataframe(
+            read_csv_files(Path(self.resourcefilepath) / "ResourceFile_Cervical_Cancer",
+                           files="parameter_values")
+        )
+
+        # note that health seeking probability quite high even though or =1
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='vaginal_bleeding',
+                    odds_ratio_health_seeking_in_adults=1.00)
+        )
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='chosen_via_screening_for_cin_cervical_cancer',
+                    odds_ratio_health_seeking_in_adults=100.00)
+        )
+
+        self.sim.modules['SymptomManager'].register_symptom(
+            Symptom(name='chosen_xpert_screening_for_hpv_cervical_cancer',
+                    odds_ratio_health_seeking_in_adults=100.00)
+        )
+
+
+    def initialise_population(self, population):
+        """Set property values for the initial population."""
+        df = population.props
+        rng = self.rng
+        p = self.parameters
+
+        # defaults
```

Comment:
no default given for `ce_date_cin_removal`

Reply:
please ensure properties are addressed in a consistent order (declaration, init, and on_birth), which help guard against ommissions